### ISSUE TITLE
Subexamples that fail cause duplicate failures

### ISSUE THREAD

**Issue by GoogleCodeExporter** [etorreborre/specs#50]
```
What steps will reproduce the problem?
$> cat myTest.scala
import org.specs._

object mySpec extends Specification {
  "my world" should {
    "be free of evil" >> {
      "giving peace" >> {false must beTrue}
      "giving prosperity" >> {true must beTrue}
    }
  }
}

mySpec.main(args)

$> scala -cp ~/Download/specs-1.4.1.jar myTest.scala
Specification "mySpec"
  my world should
  x be free of evil
    the value is false ((virtual file):10)
    x giving peace
      the value is false ((virtual file):10)
    + giving prosperity

Total for specification "mySpec":
Finished in 0 second, 49 ms
2 examples, 2 expectations, 2 failures, 0 error
What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?
Specs 1.4.1
Scala 2.7.2

Please provide any additional information below.
I have not switched to Specs 1.4.2 and Scala 2.7.3 because of limited
bandwidth. If this issues has been fixed in the newer versions, it can be
ignored.
```

Original issue reported on code.google.com by `harshad...@gmail.com` on 17 Jan 2009 at 10:43

**Comment by GoogleCodeExporter** [comment]
```
Thanks for reporting this issue, I fixed it in specs 1.4.3-SNAPSHOT.
```

Original comment by `etorrebo...@gmail.com` on 21 Jan 2009 at 3:25
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `etorrebo...@gmail.com` on 25 Feb 2009 at 9:37
- Changed state: **Verified**
